vochain/indexer/bench_test.go

```diff
@@ -59,28 +38,31 @@ func benchmarkIndexTx(b *testing.B) {
 	err := s.Generate()
 	qt.Assert(b, err, qt.IsNil)
 
+	b.ReportAllocs()
+	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
 		for j := int32(0); j < 2000; j++ {
 			vote := &state.Vote{
 				Height:      uint32(util.RandomInt(10, 10000)),
 				ProcessID:   pid,
 				Nullifier:   util.RandomBytes(32),
-				VotePackage: []byte("{[\"1\",\"2\",\"3\"]}"),
+				VotePackage: []byte(`{"votes": [1,0,1]}`),
```

Comment:
The correct way is now `vp, err := state.NewVotePackage([]int{1, 0}).Encode()`
However, there are more benchmarks that require this change, so I'll open an issue to make it at once.